protocolport/protocol_port.go

```diff
@@ -0,0 +1,7 @@
+package protocolport
+
+type ProtocolPort struct {
+	IngressPort string `json:"ingressPort" yaml:"ingressPort"`
+	LBPort      string `json:"lbPort" yaml:"lbPort"`
+	Protocol    string `json:"protocol" yaml:"protocol"`
```

Comment:
We discussed it. No strong type, as this is more of a label. The name is unfortunate.